central/detection/service/service_impl_test.go

```diff
@@ -543,6 +564,9 @@ func TestParseList_Success(t *testing.T) {
 
 	_, _, err = getObjectsFromYAML(openshiftDeploymentConfigYaml)
 	require.NoError(t, err)
+
+	_, _, err = getObjectsFromYAML(cronYaml)
```

Comment:
nit: could rewrite this to a loop instead of copy-paste, but don't think it's critical.
```go
yamlToTest := []string{listYAML, openshiftDeploymentConfigYaml, multiYaml, openshiftDeploymentConfigYaml, cronYaml}

for _, yaml := range yamlToTest {
        _, _, err := getObjectsFromYAML(yaml)
        require.NoError(t, err)
} 
```